src/graphql/syncDataSilos.ts

```diff
@@ -717,7 +717,12 @@ export async function syncDataSilo(
                 ({
                   name: key,
                   description,
-                  categories,
+                  categories: !categories
+                    ? undefined
+                    : categories.map((category) => ({
+                        ...category,
+                        name: category.name || '',
```

Comment:
Just curious, is the use of || over ?? intentional here?

Reply:
not intentional - but i dont think it changes anything here

Reply:
Agreed, just making sure!